pkg/util/vsphere/vm/power_state_test.go

```diff
@@ -110,10 +110,10 @@ func powerStateTests() {
 
 		BeforeEach(func() {
 			ctx = suite.NewTestContextForVCSim(builder.VCSimTestConfig{})
-			mgdObj = vmutil.ManagedObjectFromMoRef(types.ManagedObjectReference{
-				Type:  "VirtualMachine",
-				Value: "vm-44",
-			})
+			vmList, err := ctx.Finder.VirtualMachineList(ctx, "*")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(vmList)).To(BeNumerically(">", 0))
```

Comment:
Nit: `Expect(vmList).ToNot(BeEmtpy()))` is the way we typically express this (unless you want the failure output to just have the lengths or something)